### ISSUE TITLE
React: Masthead -  search open by default - when we tries to activate the user profile button the button is not operable

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#4215]
## Detailed description

Issue: When the search popup is displayed,  when we tries to activate the user profile button the button is not operable. instead the search popup is getting closed when we tries to activate it. Note: user profile button is not operable only when the search popup is  displayed.

Expected : User profile button should be operable.

## Additional information

![Masthead2.gif](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/0e6189ae-7210-4ead-9d50-670e2672d9f6)

**Comment by RobertaJHahn** [comment]
The only team that has search open by default is the IBM Search team.  Their configuration does not have the profile icon active.  No teams are using this configuration option currently.

**Comment by ljcarot** [comment]
@RobertaJHahn Let's move to Jan release.
